src/sentry/relay/config.py

```diff
@@ -36,53 +37,50 @@ def get_project_config(project, org_options=None, full_config=True, for_store=Fa
     :param full_config: True if only the full config is required, False
         if only the restricted (for external relays) is required
         (default True, i.e. full configuration)
-    :param for_store: If set to true, this omits all parameters that are not
-        needed for Relay. This is a temporary flag that should be removed once
-        store has been moved to Relay. Most importantly, this avoids database
-        accesses.
+    :param project_keys: Pre-fetched project keys for performance, similar to
+        org_options. However, if no project keys are provided it is assumed
+        that the config does not need to contain auth information (this is the
+        case when used in python's StoreView)
 
     :return: a ProjectConfig object for the given project
     """
     with configure_scope() as scope:
         scope.set_tag("project", project.id)
 
-    if for_store:
-        project_keys = []
-    else:
-        project_keys = ProjectKey.objects.filter(project=project).all()
-
-    public_keys = []
+    with Hub.current.start_span(op="fetch_keys"):
```

Comment:
This timing now seems a bit pointless, since we're no longer fetching anything.

Reply:
removed the timing